module/behaviour/skills/Getup/src/Getup.cpp

```diff
@@ -67,7 +65,7 @@ namespace module::behaviour::skills {
             EXECUTION_PRIORITY = config["EXECUTION_PRIORITY"].as<float>();
         });
 
-        fallenCheck = on<Last<20, Trigger<RawSensors>>, Single>().then(
+        on<Last<20, Trigger<RawSensors>>, Single>().then(
```

Comment:
Is there a real benefit to removing the reaction handle names? It seems to me like the thing that was causing the issues was just line 87, not the named reaction handles.

Reply:
Yeah but it looks like it's not even used anymore now right? This assignment is gone

Reply:
We should be using it, was mean't to stop it getting up twice, removing this SHOULD introduce an old problem

Reply:
Yeah neither of them are used once you remove that line. 
I don't see why the robot would try to get up again, there's only one ExecuteGetup emitted (when it takes priority). It would only run that again if it took priority again, which it would only do if it ran into that if statement in the check, which it would only do if it thought it was fallen over. 
I tested it a bit with this change and it was fine, but feel free to test it more 🤷

Reply:
@Bidski and @TrentHouliston are probably the people who would properly remember why it was added. Just one of those things, trying to understand why it was there before it's removed things

Reply:
Why would the robot try to get up again? Because its halfway through getting up and the check for falling triggers again? Our scripts are not exactly stellar in this area

Reply:
Not sure, just remember that was the issue, this was a number of years ago, It would try and get up after it got up

Reply:
While it's getting up, there is a gettingUp boolean that stopped us running into the getup check again. Maybe that is what was missing in the old version? We'll only be able to check again if we lose priority and the KillGetup is run.

Reply:
Tbh though what you're saying sounds different to what I've done in this PR. This old issue with double get ups would require disabling the check reactor, not the execute reactor. This PR stops us disabling the execute reactor.

Reply:
Or maybe that was broken odometry @tayloryoung6396 😝

Reply:
I mean.. it's possible, but i feel like past us would have thought of that.. this PR removes both checks.. but if it works then it works.. we'll find out soon if it breaks things again

Reply:
fallenCheck isn't used to enable/disable even though it's there, this PR only stops us disabling/enabling the getUp